scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -102,6 +102,16 @@
     env_aware=True,
 )
 
+define_config_option(
+    __monitor__,
+    "first_time_old_container_window",
+    "Optional (defaults to 0). For the first container check, how old a dead container can be and still have its "
```

Comment:
Let's rephrase this documentation to: "By default, the Scalyr Agent does not collect the logs from any pods stopped before the agent was started.  To override this, set this parameter to the number of seconds the agent will look in the past (before it was started).  It will collect logs for any pods that was started and stopped during this window."

Note, I wrote the comment assuming the window compares the pod start time, which I believe is the case.  If a pod is started before this window but stopped during the window, I believe we still won't collect the logs.  Which is fine, we just have to be clear about it.